lib/api/client-commands.js

```diff
@@ -492,10 +492,16 @@ module.exports = function(client) {
         typeString = 'browser';
       }
       return this.getLogTypes(function (types) {
-        var isAvailable = types.indexOf(typeString) >= 0;
-        if (callback) {
-          callback.call(self, isAvailable);
-        }
+        try {
+          var isAvailable = types.indexOf(typeString) >= 0;
+          if (callback) {
+            callback.call(self, isAvailable);
+          }
+        } catch (exception) {
+          if (callback) {
+            callback.call(self, false);
+          }
+        }  
```

Comment:
You could more elegantly write this as:

```js
var isAvailable;
try {
 isAvailable = Array.isArray(types) && types.indexOf(typeString) >= 0;
} catch (exception) {
  isAvailable = false; 
}

if (callback) {
  callback.call(self, isAvailable);
}
```

This also ensures the TypeError isn't thrown because there is a type check for array there.